specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -27,20 +27,142 @@ These are the global settings for the Machine Learning Services API.
 
 ``` yaml
 openapi-type: arm
-tag: package-preview-2025-04-01
+tag: package-preview-2025-04
 ```
 
-### Tag: package-preview-2025-04-01-preview
+### Tag: package-preview-2025-04
 
-These settings apply only when `--tag=package-preview-2025-04-01-preview` is specified on the command line.
+These settings apply only when `--tag=package-preview-2025-04` is specified on the command line.
 
-```yaml $(tag) == 'package-preview-2025-04-01-preview'
+```yaml $(tag) == 'package-preview-2025-04'
 input-file:
   - Microsoft.MachineLearningServices/preview/2025-04-01-preview/machineLearningServices.json
   - Microsoft.MachineLearningServices/preview/2025-04-01-preview/mfe.json
   - Microsoft.MachineLearningServices/preview/2025-04-01-preview/registries.json
   - Microsoft.MachineLearningServices/preview/2025-04-01-preview/workspaceFeatures.json
   - Microsoft.MachineLearningServices/preview/2025-04-01-preview/workspaceRP.json
+suppressions:
+  - code: AvoidAdditionalProperties
+    reason: This is necessary to allow users to specify custom inference parameters and 
+      fine-tuning hyperparameters for any model. Enforcing typecasting would require modifying 
+      contracts for each new addition by model providers. A similar approach has been previously 
+      permitted for the FinetuningJob.
+    where:
+      - $.definitions.TeacherModelSettings.properties.teacherModelInferenceParameters
+      - $.definitions.FinetuningDetails.properties.hyperParameters
+      - $.definitions.DistillationJob.properties.outputs
+  - code: DeleteResponseCodes
+    reason: Existing API behavior in 2024-10-01-preview.
+    where:
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/inferencePools/{inferencePoolName}"].delete
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/inferencePools/{poolName}/endpoints/{endpointName}"].delete
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/inferencePools/{poolName}/groups/{groupName}"].delete
+  - code: PatchIdentityProperty
+    reason: Existing API behavior in 2024-10-01-preview.
+    where:
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/inferencePools/{poolName}/endpoints/{endpointName}"].patch.parameters[6]
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/inferencePools/{poolName}/groups/{groupName}"].patch.parameters[6]
+  - code: PathForResourceAction
+    reason: Existing API behavior in 2024-10-01-preview.
```

Comment:
Surely you must fix the 'PathForResourceAction' violations for GA, if it was only an issue in previews. This could be a blocker to proper RBAC action support.

Reply:
@rkmanda Is it okay to get a temporary exception on this? I would think not, since this is going to be a GA version needing forward+backward compatibility, right?